src/lib.rs

```diff
@@ -2,13 +2,8 @@
 //! A set of specifications of crypto primitives, written in hacspec-rust
 //! See https://crates.io/crates/hacspec for details on hacspec-rust.
 
-// TODO: can we move this?
```

Comment:
I opened issue #4 to keep track of this one.

Reply:
Ah that TODO got accidentally removed as I tried to get rid of it – but the clippy lint shows up on the attributes. It might need to be suppressed in the attribute definition.